lib/gc/Transforms/GPU/LinalgToXeGPU.cpp

```diff
@@ -783,6 +985,189 @@ loadNdDescTiles(PatternRewriter &rewriter, Location loc, ValueRange loadTiles,
   return loadVec;
 }
 
+// Load from scatter 1D descriptors and return a vector of 2D tiles
+// with the shape of 'tileShape'.
+static SmallVector<Value>
+loadScatterDescTiles(PatternRewriter &rewriter, Location loc,
+                     ValueRange loadTiles, xegpu::CachePolicyAttr hint,
+                     ArrayRef<int64_t> tileShape,
+                     std::optional<VnniConfig> vnniConf = std::nullopt,
+                     DenseI64ArrayAttr transpose = nullptr,
+                     IntegerAttr transpose_bit = nullptr) {
+  int64_t elementsPerLoad = 32;
```

Comment:
is this (in other places as well) a compile-time constant?

Reply:
well I can make it a compile-time constant for now, but I guess this value depends on the hardware and should be read from somewhere (so at the end it's not a compile-time)